src/mig/modules/netstat/netstat_darwin.go

```diff
@@ -62,8 +64,104 @@ func HasSeenMac(val string) (found bool, elements []element, err error) {
 	return
 }
 
+func parseEndpointString(str string) (ip net.IP, port int) {
+	re, err := regexp.Compile("^(.*?)(%[a-z0-9]+)?\\.(\\*|[0-9]+)$")
```

Comment:
You could define it as a global variable, since the regexp is static:

```go
package main

import (
	"fmt"
	"regexp"
)

var re = regexp.MustCompile("^$")

func main() {
	if re.MatchString("test") {
		fmt.Println("test doesn't match")
	}
	if re.MatchString("") {
		fmt.Println("empty string matches")
	}
}
```